src/aws/bin/get_ngrok_url.py

```diff
@@ -0,0 +1,25 @@
+from urllib.error import URLError
```

Comment:
They will want this in bash 😅 we don't have python locally 🤷‍♂️😄

Reply:
We also don't have `ngrok`. The idea in this PR is very good, we need to discuss about it.

Reply:
Thanks @lunika :)
As we discussed, we can use a docker image to run all this.
I don't have enough time for now to manage this, but I'll take care of it asap.
Should we close this PR, and start a fresh one ?